### PR TITLE
Documentation for how to write strongbox-web-integration-tests

### PR DESCRIPTION
**Pull request by anki2189**
#https://github.com/strongbox/strongbox/issues/1545

Before merging this PR https://github.com/strongbox/strongbox-web-integration-tests/pull/59 should be merged.

### CODE REVIEW

File: docs/developer-guide/writing-web-integration-tests.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+For this to work, you need to have installed `docker` and `docker-compose` on your machine.
+
+Afterwards, just go to the respective sub-project (i.e. `cd maven`) and execute `docker-compose up`.
+
+Each module has a `docker-compose.yml` file.
+
+You can run all the tests (for all modules) like this:
+
+```
+for tool in `ls -ap | grep \/ | grep -v -e "\."`; do cd ${tool} && docker-compose up & cd -; done
```
Comment (steve-todorov) [inline docs/developer-guide/writing-web-integration-tests.md]: This command will result in multiple containers outputting logs into the console at the same time, which makes it impossible to follow what's actually happening.  We can improve the original command with this one:  

```suggestion
find . -maxdepth 2 -type f -name "docker-compose.y*ml" -exec docker-compose -f {} up \;
```

Comment (anki2189) [reply docs/developer-guide/writing-web-integration-tests.md]: Fixed

Comment (steve-todorov) [reply docs/developer-guide/writing-web-integration-tests.md]: Thanks! :)

**Comment by carlspring** [comment]
Thank you very much for this pull request!

**Pull request merged**